### ISSUE TITLE
Inconsistent definition of minimal config.yaml

### ISSUE THREAD

**Issue by ElliotSilver** [FHIR/sushi#559]
While attempting to track down a problem, I commented out the complete contents of my config.yaml. When I run sushi, I got the message: `error SUSHI minimally requires the following configuration properties to start processing FSH: id, version, canonical, fhirVersion.`

If I add those properties back in, I get:
```
error Configuration missing required property: name
  File: fsh/config.yaml
error Configuration missing required property: status
  File: fsh/config.yaml
error Configuration missing required property: copyrightYear
  File: fsh/config.yaml
error Configuration missing required property: releaseLabel
  File: fsh/config.yaml

```
It is unclear what the minimal config.yaml file is.

**Comment by cmoesel** [comment]
Hi @ElliotSilver.  You are right!  We actually discovered and fixed this last week (d4cd799) but we haven't yet released it.  We expect to release a new version of SUSHI in the next couple of days that will contain this fix.

(To be clear: the fix is to improve the error message to correctly list all required properties.  The set of required properties will differ, however, based on whether or not you have the FSHOnly flag set.  If you want SUSHI to create an IG, then you will need those additional properties listed above as well.

**Comment by cmoesel** [comment]
Fixed in SUSHI 0.15.1.
